modules/core/shared/src/main/scala/lucuma/core/math/validation/MathValidators.scala

```diff
@@ -17,6 +17,9 @@ trait MathValidators {
   val epoch: InputValidSplitEpi[Epoch] =
     InputValidSplitEpi.fromPrism(Epoch.fromString, "Invalid epoch")
 
+    val epochNoScheme:InputValidWedge[Epoch] = 
```

Comment:
Is it miss indented?

Reply:
Oh, probably. I need to re-enable scalaFmt for me locally.